src/anime_graph.cpp

```diff
@@ -59,8 +87,11 @@ Edge* AnimeGraph::getEdge(unsigned first, unsigned second) const {
     return node_list.at(first)->edges.at(second);
 }
 
-// Return the node with the largest # of members. If two shows have the same # of members, the better rated shows gets taken.
-// If the map is empty, returns NULL
+/**
+ * @brief returns the node with the most members, or better rated if tie
+ * 
+ * @return Node* 
```

Comment:
What you have in the brief is what you would write here. You can explain how the biggest is found if you want in the brief.